README.md

```diff
@@ -101,9 +101,23 @@ Visit `localhost:3001/api-docs` to see Swagger UI for API endpoints 📑
 
 ## Running tests
 
+### API
+
 - `bundle exec rspec` or `bundle exec guard` to watch
 - When tests pass and you're ready for a PR, please run `rails rswag` to update the API documentation
 
+### Frontend
+
+- `yarn test` (auto-watch) or `yarn test-once` to run the suite one time only
+
+### End to End
+
+- `yarn cy:ci` from the root directory
```

Comment:
@katelovescode Can we also add a step to build the frontend and copy it to the `public` directory before running `yarn cy:ci`?